scripts/build_scenario1.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+from jinja2 import Environment, FileSystemLoader
+
+# Définir le répertoire contenant le template
+template_dir = '/home/chausser/test/'
+
+
+
+# Configuration de Jinja2 pour charger le template
+env = Environment(loader=FileSystemLoader(template_dir))
+template = env.get_template('scenario1.yaml.j2')
+for missing in range(1,7):
+    for width in range(1,9):
+        for depth in range(2,11):
+            name = 'scenario_1_depth'+str(depth)+'_width'+str(width)+'_missing'+str(missing)
+            # Les données à insérer dans le template
+            missing_rate = 0.1*missing
+            missing_rate = round(missing_rate,3)
+            data = {
+                'meta': {
+                    'language': 'en'
+                },
+                'variants': {
+                    'letters': {
+                        'mispell': {
+                            'law': 'Bernouilli',
+                            'rate': 0.5,
```

Comment:
these also should be variables on top of the script -> we want to be able to change at first glance the parametrization of the scenario